db/data/regions/CN/en.yml

```diff
@@ -8,10 +8,12 @@ en:
             default: Full address
             optional: Full address (optional)
           errors:
-            blank: Enter a full address
+            blank_instructional: Enter a full address
+            blank_informative: Full address is missing
```

Comment:
Is this a field-level validation? Or does it appear when a single element is missing from a form?